.flowconfig

```diff
@@ -6,8 +6,9 @@ module.name_mapper='^firefox-profiler\/\(.*\)$' -> '<PROJECT_ROOT>/src/\1'
 module.name_mapper.extension='ftl' -> '<PROJECT_ROOT>/src/types/mocks/ftl.js'
 
 [ignore]
-.*node_modules/.*
-res/before-load.js
+<PROJECT_ROOT>/node_modules/.*
+<PROJECT_ROOT>/res/before-load\.js
+<PROJECT_ROOT>/dist/.*
```

Comment:
Oops, looks like other PRs changes are also here. Could you revert them?

Reply:
oops